.github/workflows/ruff.yml

```diff
@@ -15,16 +15,11 @@ jobs:
   ruff:
     name: ruff
     runs-on: ubuntu-latest
+
     steps:
-      - name: Checkout
-        uses: actions/checkout@v4
-      - name: Set up Python
-        uses: actions/setup-python@v4
-        with:
-          python-version: "3.10"
-          cache: 'pip'
-      - run: |
-          python -m pip install --upgrade pip
-          pip install ruff
-      - name: Run Ruff
-        run: ruff .
+    - uses: actions/checkout@v4
+
+    - run: python -Im pip install --user ruff
```

Comment:
Is this a typo `python -Im pip install --user ruff` should it be `python -m pip install --user ruff`?

Reply:
No that's intentional. It's python's isolated mode flag

https://docs.python.org/3/using/cmdline.html#cmdoption-I

Reply:
Noted, with thanks. Let me approve this.